pkg/model/dev.go

```diff
@@ -555,6 +555,17 @@ func (b *BuildInfo) setBuildDefaults() {
 	}
 }
 
+func (b *BuildInfo) setDockerfileDefaults() {
+	if b.Context == "" {
+		return
+	}
+	if b.Dockerfile == "" {
```

Comment:
@jLopezbarb Does dockerfile intended to support relative paths, absolute paths or can it contain  any of them?